<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWnd</class>
 <widget class="QMainWindow" name="LoginWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheckEVD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="username_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>以管理员身份登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginBtn</sender>
   <signal>clicked(bool)</signal>
   <receiver>LoginWnd</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>username_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
 </slots>
</ui>
